snap/snapcraft.yaml

```diff
@@ -51,14 +51,11 @@ parts:
         - libssl-dev
         - libpq5
         - postgresql-server-dev-all
-    configflags:
+    cmake-parameters:
         - -DFLB_DEBUG=On
         - -DFLB_OUT_KAFKA=On
         - -DFLB_JEMALLOC=On
         - -DFLB_EXAMPLES=OFF
         - -DFLB_SHARED_LIB=Off
         - -DFLB_OUT_PGSQL=On
-
-layout:
-  /etc/fluent-bit:
-    bind: $SNAP/etc/fluent-bit
+        - -DFLB_CONFIG_YAML=Off
```

Comment:
Should we disable YAML support? It needs some more dependencies but anecdotally `libpq` is much worse for that with triggering cves.

I think we just need `libyaml-dev` in here to get the benefits of using the new config format. It might be worth it for the eventual transition to YAML.

Reply:
I disable it, because the process was not working, but I can definitely try it with `libyaml-dev`in July (holidays for 2 weeks)

Reply:
Yeah I think it makes sense to align with the other builds if we can. 
If there is some reason it does not work then of course we can take it but should test at least.